src/data/users/promise.json

```diff
@@ -29,5 +29,6 @@
       "projectDescription": "Developed with React and Laravel",
       "projectLink": "https://quickrace.com.ng"
     }
-  ]
+  ],
+  "id": 5
```

Comment:
here too

Reply:
Gives undefined, thus, when two profiles are not set, one overrides the other, fixed this on my pc and would make a push for this to this pr instead